index.html

```diff
@@ -0,0 +1,145 @@
+          </a>
+        <div id="nav-pages">
+          <a class="pop-window" href="/about.html">About</a>
+          <a class="pop-window" href="#about">Program</a>
+          <a class="pop-window" href="#footer"> Join</a>   
+          <a class="pop-window" href="#footer"> Sponsors</a>  
+          <a class="pop-window" href="#footer"> News</a>  
+        </div>
+        <div id="contact"> 
+          <button id="contact-btn" > Contact </button>
+        </div>
+      </div>
+    </nav>
+    <div id="hero-bg"> 
+    <div class="title">      
+      <h1> <span class="margin-none">Music is life!</span> <br> GET READY FOR THE ULTIMATE FESTIVAL WINTER 2023</h1>
+    </div>
+    <div class="paragraph"> Lolla takes over historic Grant Park in Downtown Chicago, alongside the beautiful Chicago skyline and the shores of Lake Michigan. Take advantage of interactive areas 
+      with giveaways, merch and more!</div>
+    <div class="date"><h2>SAVE THE DATE! MARCH 2-5-2023 </h2>
+      <p>@ Chicago, Illinois, USA.</p>
+    </div>
+  </div>
+  </div>
+</section>
+
+<!-- SECTION MAIN PROGRAM -->
+
+<section id="main-program">
+  <div id="program">
+    <div id="program-title" class="flex-center">
+    <h2>Schedule <div id="program-separator-line" class="guide-bar"></div> </h2>
+    </div>
+    <div class="cards"> 
+      <div class="program-cards">
+      <img src="/bg-img/music.png" alt="icon">
+      <h3>DAY 1</h3>
+      <p class="card-p">Meet the bands, lets get to know eachother and chill before the music starts</p>
+      </div>
+      <div class="program-cards">
+      <img src="/bg-img/sleep-removebg-preview.png"  alt="icon">
+      <h3>DAY 2</h3>
+      <p class="card-p">Sleep on the grass and get in touch with your primitive mind. Have fun !</p>
+      </div>
+      <div class="program-cards">
+      <img src="/bg-img/img_552945.png" alt="icon">
+      <h3>DAY 3</h3>
+      <p class="card-p">Listen to music while you look directly at the sun. Please wear sunglasses</p>
+      </div>
+      <div class="program-cards">
+      <img src="/bg-img/music4-removebg-preview.png"alt="icon">
+      <h3>DAY 4</h3>
+      <p class="card-p">Listen to more music, get lost in the rithm and try not to fall sleep, please</p>
+      </div>
+      <div class="program-cards">
+      <img src="/bg-img/music5.png" alt="icon">
+      <h3>DAY 5</h3>
+      <p class="card-p">Say goodbay to all the wonderful friends you've make and promise them to keep in touch</p>
+      </div>
+    </div>
+    <div id="program-btn" class="flex-center">
+    <button id="join-btn">Join our club</button>
+    </div>
+    <div id="program-more" class="flex-center">
+      <a><h2> See the whole program <div id="program-separator-line" class="guide-bar"></div></h2> </a>
+      </div>
+  </div>
+</section>
+
+<!-- JAVASCRIPT-BANDS -->
+
+<section id="bands"> 
+
+ </section>
+
+<section id="section-4">
+  <div id="about-bg-4"> 
```

Comment:
- [optional] kindly ensure that your mobile menu disappears when switched to the desktop version.
![Screenshot (549)](https://user-images.githubusercontent.com/65298048/197230042-e310dfc9-49de-40be-b151-e02de48092e1.png)